bedrock/firefox/templates/firefox/whatsnew/whatsnew-50.html

```diff
@@ -0,0 +1,36 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/sendto" "download" %}
+
+{% extends "firefox/mobile-download-desktop.html" %}
+
+{% block page_css %}
```

Comment:
Do we need to add a `{% block gtm_page_id %}data-gtm-page-id="/firefox/whatsnew/"{% endblock %}` block for this page?